### ISSUE TITLE
Training and validation accuracy of models

### ISSUE THREAD

**Issue by martin-kirilov** [Alex-Fabbri/Multi-News#22]
Hi Alex,

I've been training the transformer model from OpenNMT on your dataset, as explained in another issue. I was wondering if I am doing anything wrong, since the training accuracy is around **44**, and the validation accuracy is around **37** at step **20000**. Do you know what are the accuracies of the models you provided (e.g. I can see that your transformer model is also trained for 20000 steps)?

**Comment by Martinski95** [comment]
I have also generated some summaries using the two models (my transformer at 20000 steps and the your pre-trained model), and they are quite different in terms of quality and factual accuracy.
